packages/typescript-estree/src/parseSettings/warnAboutTSVersion.ts

```diff
@@ -38,11 +40,12 @@ export function warnAboutTSVersion(
     const border = '=============';
     const versionWarning = [
       border,
-      'WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.',
+      'WARNING: @typescript-eslint/typescript-estree does not support your TypeScript version.',
       'You may find that it works just fine, or you may not.',
       `SUPPORTED TYPESCRIPT VERSIONS: ${SUPPORTED_TYPESCRIPT_VERSIONS}`,
       `YOUR TYPESCRIPT VERSION: ${ACTIVE_TYPESCRIPT_VERSION}`,
       'Please only submit bug reports when using the officially supported version.',
+      `Your @typescript-eslint/typescript-estre version: ${TYPESCRIPT_ESTREE_VERSION}`,
```

Comment:
Also, the two existing package prints are next to each other and in all caps. This one is elsewhere. Any particular reason?

Reply:
I understand your objection. Here is my thought:
1. the current printed version are related to typescript so, 
from my point of view the typescript-estree version is an additional information that is not strictly related to the current typescript version and the supported typescript versions range.
2. I thought to add it after the bug report note since it can be useful when someone wants to open an issue.

As an alternative we can do something like ESLint where version is printed right after the error?

```text
You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

@typescript-eslint/typescript-estree: ${TYPESCRIPT_ESTREE_VERSION}

You may find that it works just fine, or you may not.'

SUPPORTED TYPESCRIPT VERSIONS: ${SUPPORTED_TYPESCRIPT_VERSIONS}

YOUR TYPESCRIPT VERSION: ${ACTIVE_TYPESCRIPT_VERSION}

Please only submit bug reports when using the officially supported version.
```

---

No problem adding a 

```text
ESLINT TYPESCRIPT ESTREE VERSION: ${ACTIVE_TYPESCRIPT_VERSION}
```
after or before typescript range and current version  trough 😅

Reply:
I think the messages being printed now are kind of spread apart. More visual size makes people less likely to read them ("eww, so much log spam!"). How about...

```plaintext
You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

* @typescript-eslint/typescript-estree version: ${TYPESCRIPT_ESTREE_VERSION}
* Supported TypeScript versions: ${SUPPORTED_TYPESCRIPT_VERSIONS}
* Your TypeScript version: ${ACTIVE_TYPESCRIPT_VERSION}

Please only submit bug reports when using the officially supported version.
```

WDYT?

cc @bradzacher as historian - I don't personally love all caps in messages. I'm guessing they're there because people used to mess this up a lot, especially before versions were synchronized. I wonder if we can lowercase them now?

Reply:
I like this format! 
Placing a bulleted list between two paragraphs helps capture the reader's focus.

Reply:
This log is from the pre-unification days IIRC. It dates back to the stand-alone package typescript-estree days that James and Armano lead.

I assume it was just added for emphasis.